spec/.rubocop.yml

```diff
@@ -75,9 +75,17 @@ RSpec/ExpectInHook:
   Enabled: false
 
 # your naming scheme is not in possession of all the facts
-RSpec/FilePath:
+
+RSpec/IndexedLet:
+  Enabled: false
+
+RSpec/SpecFilePathFormat:
   Enabled: false
 
+RSpec/SpecFilePathSuffix:
+  Enabled: true
+
+
```

Comment:
I believe there should be only one line here?

```suggestion

```

Reply:
fixed with rebased changes